### PR TITLE
Fix RandomBrightness for Ragged Tensor

### LINKED ISSUE #1494: Random Brightness raises an error when the image is Ragged tensor
Similar to #1481.RandomBrightness is not included in keras_cv/layers/preprocessing/ragged_image_test.py.

@LukeWood @ianstenbit  I can open a PR to fix this

### PR DESCRIPTION
**Pull request by soma2000-lang**
# What does this PR do?

Fixes #1494

## Before submitting
- 
- [x] Did you read the [contributor guideline](https://github.com/keras-team/keras-cv/blob/master/.github/CONTRIBUTING.md),
      Pull Request section?
- [x] Was this discussed/approved via a Github issue? Please add a link
      to it if that's the case.
- [ ] Did you write any new necessary tests?

## Who can review?

Anyone in the community is free to review the PR once the tests have passed. Feel free to tag
members/contributors who may be interested in your PR.

### CODE REVIEW

**Review by LukeWood — APPROVED** [review]

**Comment by LukeWood** [comment]
@soma2000-lang looks like this fails the lint.  Please run ./shell/format.sh

**Review by LukeWood — CHANGES_REQUESTED** [review]
Ah wait - unit tests actually fail.  

Please address all unit test failures before re-requesting review!

File: keras_cv/layers/preprocessing/random_brightness.py [hunk 1]
```diff
@@ -80,9 +80,17 @@ def get_random_transformation_batch(self, batch_size, **kwargs):
         return random_rgb_deltas
 
     def augment_ragged_image(self, image, transformation, **kwargs):
```
Comment (james77777778) [inline keras_cv/layers/preprocessing/random_brightness.py:82]: Following should fix the issue:

```python
def augment_ragged_image(self, image, transformation, **kwargs):
    image = tf.expand_dims(image, axis=0)
    transformation = tf.expand_dims(transformation)
    image = self.augment_images(
        images=image, transformations=transformation, **kwargs
    )
    return tf.squeeze(image, axis=0)
```

**Comment by LukeWood** [comment]
Thank you @soma2000-lang !

**Review by LukeWood — APPROVED** [review]

**Pull request merged**